CONTRIBUTING.md

```diff
@@ -28,6 +28,14 @@ And this list will only grow over time.
 
 Please read the [Code of Conduct][coc]. It's adapted from the commonly used contributor covenant, but it's not just boilerplate. Be sure you can uphold the project's shared values and abide by the standards laid out in that document. You're more than welcome to ask for clarification.
 
+These values will have a direct impact on the features we do and don't implement, how we prioritize them, and how they are designed. For example:
+
+* We believe tracking and mass data harvesting are dangerous and unethical; we won't build or support those activities and will attempt to limit the personal data we collect
+* We believe that cryptocoins, NFTs, and related technologies are purposefully wasteful and deceptive; we will not incorporate them
+* We believe so-called AI—especially generative AI—is extractive, wasteful, hazardous, and usually deceptive; we will view it with skepticism
+* We will prefer opt-in rather than opt-out dynamics wherever possible
```

Comment:
Might be useful to call out consent practices here to clarify what this means. I do not think a popup before viewing a page effectively communicates consent for example but it *is* technically an opt-in.

Reply:
```suggestion
* We will prefer opt-in rather than opt-out dynamics wherever possible
* We will seek and respect consent, by helping people to make well informed choices without manipulation
```

Does that cover what you had in mind?

Reply:
Perfect! I love it.